scripts/build_validation_models.sh

```diff
@@ -11,38 +11,38 @@ function try_build_project {
     typeset -x CMAKE_PREFIX_PATH="$ns_install_path:$CMAKE_PREFIX_PATH"
 
     if [ -e "$build" -a ! -d "$build" ]; then
-	echo "File exists at '$build', skipping."
-	return 1
+    echo "File exists at '$build', skipping."
+    return 1
     fi
```

Comment:
Replacing the tabs is great, but they'll need to be replaced with eight spaces instead of four in order to preserve indenting.

Reply:
I must have missed a few, I will fix.